src/Static.js

```diff
@@ -103,29 +115,29 @@ class Static extends React.Component {
         var ret = await chrome.webview.hostObjects.scriptObject.SignOut();
         this.setState({
           signInStatus: !ret,
-          signInTitle: "Sign In"
-        })
+          signInTitle: 'Sign In'
+        });
       }
       else {
-        var btn = document.getElementById("btnSignIn");
+        var btn = document.getElementById('btnSignIn');
         btn.classList.add('disableButton');
         btn.disabled = true;
 
-        this.setState({ signInTitle: "Signing In" })
-        var ret = await chrome.webview.hostObjects.scriptObject.SignIn();
-        this.setState({ signInStatus: ret });
+        this.setState({ signInTitle: 'Signing In' });
+        var ret2 = await chrome.webview.hostObjects.scriptObject.SignIn();
+        this.setState({ signInStatus: ret2 });
 
         btn.classList.remove('disableButton');
         btn.disabled = false;
         if (ret) {
```

Comment:
Does `ret` needs to be renamed to `ret2` here as well?

Reply:
I've changed the name because eslint was giving errors. It was changed to `status`, a better term. Thanks!